client/src/components/List/List.jsx

```diff
@@ -0,0 +1,41 @@
+import React, {useState} from 'react'
+import Data from '../../data/businesses'
+import './List.css';
+export default function List() {
+//state for JSON data
+const [data, setData] = useState(Data);
+
+    return (
+        <div >
+        <h1>List View</h1>
+            <div id="location-container">
+            {/* Mapping through data to display on each bizcard */}
+            {data.map((biz, index)=>( 
+            <div id="location-cell">
+            <img id="business-image" src="https://media.timeout.com/images/105393878/image.jpg"/> <br/>
+             <span id="location-name">{biz.name}</span> <br/>
+             <img id="location-png" />
+            {biz.address_city} <br/>
+            <img id="location-png" />
+             {biz.address_street} <br/>
```

Comment:
Since we have city here, I think we can remove address and just leave that for when they click on card.

Reply:
👍